firewood/src/merkle.rs

```diff
@@ -34,13 +34,15 @@ pub enum MerkleError {
     ParentLeafBranch,
     #[error("removing internal node references failed")]
     UnsetInternal,
+    #[error("error updating nodes with: {0}")]
```

Comment:
nit: remove "with"; adds no value and might make the sentence grammatically awkward!